simcase-game/src/components/Navbar.js

```diff
@@ -110,73 +71,16 @@ function Navbar({ showDialogValue = false, modalValue = null }) {
   return (
     <Header css={container}>
       <button onClick={openAbout} css={about}>
-        <svg viewBox="0 0 512 512">
```

Comment:
The lines of those svg are in the file Symbols and define their id's to be called like `<use xlinkHref="#info" />`